File: 2020/rust/x/src/main.rs
use std::time::Instant;

fn main() {
    let file_contents = include_str!("../../../input/x.txt");

    let start = Instant::now();
    let ans_one = part_one(file_contents);
    let duration = start.elapsed();
    let expected_one = 0;
    println!(
        "Part one answer: {}, expected {}. Completed in: {:?}",
        ans_one, expected_one, duration
    );
    assert_eq!(ans_one, expected_one);

    let start = Instant::now();
    let ans_two = part_two(file_contents);
    let duration = start.elapsed();
    let expected_two = 0;
    println!(
        "Part two answer: {}, expected {}. Completed in: {:?}",
        ans_two, expected_two, duration
    );
    assert_eq!(ans_two, expected_two);
}

fn part_one(file_contents: &str) -> i32 {
    file_contents.len() as i32
}

fn part_two(file_contents: &str) -> i32 {
    file_contents.len() as i32
}

File: 2020/js/x/main.js
const assert = require('assert')
const { getDayInputAsLines } = require('../lib/utils')

async function getInput () {
  return getDayInputAsLines(0)
}

function partOne (input) {
  return input.length
}

function partTwo (input) {
  return input.length
}

(async function run () {
  const input = await getInput()
  console.time('part 1 duration')
  const answerOne = partOne(input)
  console.timeEnd('part 1 duration')
  console.log('part 1 answers. expected: TBC, actual:', answerOne)
  assert.equal(input.length, 0)

  console.time('part 2 duration')
  const answerTwo = partTwo(input)
  console.timeEnd('part 2 duration')
  console.log('part 2 answers. expected: TBC, actual:', answerTwo)
  console.log('part 2 answers. expected: TBC, actual:', answerTwo)
  assert.equal(input.length, 0)
}())
